swagger/smart_transfer_protocol.yaml

```diff
@@ -0,0 +1,230 @@
+openapi: 3.0.3
```

Comment:
Instead of `swagger/` path let's have:
- `protocols/smart-transfer-protocol.openapi.yaml`
- `protocols/smart-transfer-protocol.asyncapi.yaml`

Reply:
I reorganized the folders and names this way, except the `openapi` part, which needs to be written from scratch